lib/node_modules/@stdlib/array/fixed-endian-factory/README.md

```diff
@@ -357,6 +357,26 @@ var bool = arr.every( isNegative );
 // returns true
 ```
 
+
+<a name="method-some"></a>
```

Comment:
This documentation is inserted in the incorrect spot and also breaks the documentation of `every`. See `array/bool` for an example of `some` docs: https://github.com/stdlib-js/stdlib/tree/28db4f2869749f6a6d6f7ae6476bd27f65052e96/lib/node_modules/%40stdlib/array/bool#booleanarrayprototypesome-predicate-thisarg-